.github/workflows/autorelease.yaml

```diff
@@ -0,0 +1,24 @@
+name: Autorelease
+on:
+  push:
+    branches:
+      - main
+      - release-automation
```

Comment:
what changes are inside in release-automation that are not in the main branch?

Reply:
This is only to enable verification of the PR. Removing now